import-export-cli/mi/impl/constants.go

```diff
@@ -41,3 +41,4 @@ const monthHeader = "MONTH"
 const yearHeader = "YEAR"
 const transactionCountHeader = "TRANSACTION COUNT"
 const userIDHeader = "USER ID"
+const RoleHeader = "ROLE"
```

Comment:
```suggestion
const roleHeader = "ROLE"
```
Any reason why we made this RoleHeader variable start from uppercase?
"If the functions with names that start with an uppercase letter will be exported to other packages. If the function name starts with a lowercase letter, it won't be exported to other packages, but you can call this function within the same package." [1]

[1] https://www.golangprograms.com/naming-conventions-for-golang-functions.html

Reply:
fixed in 84dfebb1a9d2419c951bfbb295afe6ccd5c8dfe2